include/inviwo/core/datastructures/geometry/decoratedmesh.h

```diff
@@ -208,8 +269,11 @@ using RadiiBuffer = BufferTrait<float, 1, static_cast<int>(BufferType::NumberOfB
  * \snippet modules/base/algorithm/meshutils.cpp Using Colored Mesh
  *
  */
+
+// DecoratedMesh<buffertraits::IndexBuffer,buffertraits::PositionBuffer> test;
+
 template <typename... BufferTraits>
-class DecoratedMesh : public Mesh {
+class DecoratedMesh : public Mesh, public BufferTraits... {
```

Comment:
Should probably assert that all BufferTraits are are derived from BufferTraits..

Reply:
I.e. ststic_assert(...j